local_postgres_docker_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
What @cgreene says is precisely what I was thinking. We definitely want to use a properly tuned, optimized and backed up server for our production DB, and we get that with RDS on AWS. At the same time, a Docker-based DB would fulfill two of the goals I think we should have:

> a developer (or independent lab) can pull our Docker image and get up and running with minimal configuration

Noted here: https://github.com/greenelab/adage-server/issues/193#issuecomment-313806195